autoinstall_templates/sample.seed

```diff
@@ -117,6 +117,10 @@ d-i pkgsel/include string ntp ssh wget
 # Note: options passed to the installer will be added automatically.
 d-i debian-installer/add-kernel-opts string $kernel_options_post
 
+# Grub settings
+d-i grub-installer/grub2_instead_of_grub_legacy boolean true
+d-i grub-installer/bootdev string /dev/[sv]da
```

Comment:
@jmaas I did not try with xen, but I have been using that option for installing KVM, VirtualBox and baremetal on a regular basis, and it works very well for me. If it does not work with xen... well, it is not going to work any less than it does without it.